portfolio/src/main/webapp/script.js

```diff
@@ -57,7 +58,7 @@ const getComments = function() {
         const commentsContainer = document.getElementById('comments-container');
         commentsContainer.innerHTML = '';
         
-        for(let i in comments) {
+        for(let i = 0; i < comments.length; i++) {
```

Comment:
This is definitely preferable over a for-in loop (which is problematic due to iterating over irrelevant properties in some cases) but I think a for-of loop (`for (let comment of comments) {`) is even more succinct since it removes the need for the variable i. But again, up to you.